modules/openapi-generator/src/main/resources/typescript-node/models.mustache

```diff
@@ -119,6 +119,9 @@ export class ObjectSerializer {
             }
             return transformedData;
         } else if (type === "Date") {
+            if (typeof data === "string") {
+                return new Date(data).toISOString();
+            }
```

Comment:
```typescript
if (!(data instanceof Date)) {
    return data;
}
```

would also work fine.

Reply:
thanks for your contribution. I would actually prefer the alternative you propose, since .toISOString would change the resulting string (appending the time) and would convert it to UTC